src/scripts/datetime.js

```diff
@@ -1,4 +1,4 @@
-import globalize from './globalize';
+import globalize from '../lib/globalize';
```

Comment:
Why not use global import when you change `./` and there is no need to preserve the style of sibling lines?